pkg/operator/ceph/config/monstore.go

```diff
@@ -93,7 +94,11 @@ func (m *MonStore) SetIfChanged(who, option, value string) (bool, error) {
 // Set sets a config in the centralized mon configuration database.
 // https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
 func (m *MonStore) Set(who, option, value string) error {
-	logger.Infof("setting option %q (user %q) to the mon configuration database", option, who)
+	if logger.LevelAt(capnslog.TRACE) {
+		logger.Tracef("setting option %q (user %q) = %q to the mon configuration database", option, who, value)
```

Comment:
This code isn't incorrect, but I don't think it's necessary to have an if-else construct. The logger itself internally handles whether or not the trace log is emitted when it checks the log level. It seems cleaner to me to emit both logs and not worry too much about having duplicates when trace logging is enabled. 

```go
logger.Tracef("insecure logs that might leak password")
logger.Infof("normal non-leaking logs")
```

I'm happy to approve the PR either way. Artem or Travis may have thoughts that differ from mine.

Reply:
the only benefit of extra log lvl check is not having 2 almost duplicate statements when trace lvl is used. 
But probably it will not make much difference.

Reply:
I would also vote to go with Blaine's suggestion, to keep the code simpler. When tracing is enabled, IMO it's perfectly normal to have lots of extra logging that may be overlapping.

Reply:
done